src/index.html

```diff
@@ -7,6 +7,381 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" 
+              href="#"
+            >
```

Comment:
please fix it everywhere
```suggestion
            <a 
              class="nav__link is-active" 
              href="#"
            >
```

Reply:
Unfortunately I can't find the border difference. I setted the same border like on the mochup. But the problem remains